sdk/js/src/token_bridge/getOriginalAsset.ts

```diff
@@ -1,3 +1,5 @@
+const sha256 = require("js-sha256");
```

Comment:
mixing import and require

Reply:
also, I'm not sure if we need this additional library - the functionality is likely provided by ethers. we should check how this is done by the cosmwasm code and stay consistent